src/reqwest_loader.rs

```diff
@@ -25,8 +26,14 @@ impl<T: Fn(&Url, RequestBuilder) -> RequestBuilder + Send + Sync + Unpin>
   ReqwestLoader<T>
 {
   pub fn new(client_builder: reqwest::ClientBuilder, middleware: T) -> Self {
+    // reqwest::ClientBuilder adds "Accept: */*" as its sole default header
+    // but we override that here to encourage content negotiation.
+    let accept_header = reqwest::header::HeaderValue::from_static("application/javascript, application/typescript, text/javascript, text/typescript, text/jsx, text/tsx, application/json;q=0.9, text/plain;q=0.8, application/octet-stream;q=0.8");
+    let mut default_headers = reqwest::header::HeaderMap::new();
```

Comment:
I feel like it's good to have `*/*;q=0.7` as a backup.

Reply:
We will error out on any content types that aren't one of the above though, right?

Reply:
I'm considering a case where a user indeed serves typescript at `example.com/path.ts,` but some server set the content type to `video/mp2t` by mistake.

If we allow `*/*`, we only error out after knowing that content at the provided path is not typescript.

Reply:
Ah, statement retracted then.

Reply:
It tend to agree `*/*` is safer, it still allows a server to negotiate the best content, but doesn't cause a hard fail.

Reply:
Added `*/*;q=0.7`